src/main/javascript/src/views/student/StudentQuiz.vue

```diff
@@ -275,7 +275,14 @@ export default {
       return isNaN(time) ? '' : moment.duration(time, "milliseconds").humanize();
     },
     currentScore() {
-      return `${this.selectedSubmission?.totalAlteredGrade} / ${this.assignmentData?.maxPoints}`;
+      let grade;
+      if (!this.selectedSubmission) {
+        grade = '-';
+      } else {
+        const { totalAlteredGrade, alteredCalculatedGrade } = this.selectedSubmission;
+        grade = totalAlteredGrade ? totalAlteredGrade : alteredCalculatedGrade;
```

Comment:
So, it's possible that totalAlteredGrade is 0 (or 0.0) and thus not null but also would evaluate to false here. I think here the code should check if totalAlteredGrade is not equal to null.

Reply:
Good catch, fixed